DEPRECATED.md

```diff
@@ -6,7 +6,9 @@ As of release 1.3.0, Envoy will follow a
 The following features have been DEPRECATED and will be removed in the specified release cycle.
 A logged warning is expected for each deprecated item that is in deprecation window.
 
-## Version 1.9.0 (pending)
+## Version 1.10.0 (pending)
```

Comment:
not sure why those appears on diff, they shouldn't, can you merge master again?

Reply:
done